### PR TITLE
Add to cart - bug fix

### PR DESCRIPTION
**Pull request by szewczykmira**
Issue #644

### CODE REVIEW

File: tests/test_cart.py [hunk 1]
```diff
@@ -287,6 +287,18 @@ def test_view_invalid_add_to_cart(client, product_in_stock, request_cart):
...
 
+def test_view_negative_add_to_cart(client, product_in_stock, request_cart):
+    variant = product_in_stock.variants.get()
+    request_cart.add(variant, 2)
+    response = client.post('/cart/add/%s/' % (variant.product_id,),
+                           {'variant': variant.pk, 'quantity': -3})
+    assert response.status_code == 302
+    assert request_cart.quantity == 2
+    location = response['Location']
+    assert location == '/products/%s-%d/' % (product_in_stock.get_slug(),
```
Comment (artursmet) [inline tests/test_cart.py]: Please use `reverse()` to resolve url's and don't hardcode paths in tests.

Comment (artursmet) [inline tests/test_cart.py]: You can also use a helper (https://github.com/mirumee/saleor/blob/master/tests/utils.py) to extract `Location` from response object. There is a difference between two django versions we support.

**Comment by patrys** [comment]
This does not help with the context. We've discussed a more robust solution today with @artursmet.

**Comment by szewczykmira** [comment]
This solution has been replaced by pull request #651

**Pull request closed without merging**